### PR TITLE
Ignore localization generation if API doesn't support it

### PR DESCRIPTION
**Pull request by EugeneElkin**

### CODE REVIEW

File: lib/VisualBuilder.js [hunk 1]
```diff
@@ -312,6 +312,13 @@ class VisualBuilder extends EventEmitter {
     }
 
     _getLocalization(visualConfig) {
+        let apiVersionGroups = visualConfig.apiVersion.split(".");
+
+        // If API version is less than 1.6.0 it will mean that API doesn't contain schema.stringResources.json
+        if (apiVersionGroups[1] < 6) {
```
Comment (ignatvilesov) [inline lib/VisualBuilder.js]: I don't think that this is nice idea to use API version. It'd be better to check file in the directory before using this file.

Comment (EugeneElkin) [reply lib/VisualBuilder.js]: As I understood we supposed to use this approach on the yesterday meeting. Correct me if I wrong.

Comment (uve) [reply lib/VisualBuilder.js]: please check the file in directory instead

File: lib/VisualBuilder.js [hunk 2]
```diff
@@ -312,22 +312,28 @@ class VisualBuilder extends EventEmitter {
...
+                        this._getValidationSchema(stringResourcesPath),
                         this._getLocaleFromStringResources(localeResourcesFiles)
                     ])
                         .then(([schema, localeStringJsons]) =>
                             Promise.all(localeStringJsons.map(localeJson => this._validateJsonBySchema(localeJson, schema))));
                 }
 
                 return Promise.all([
-                    this._getValidationSchema(this.visualPackage.buildPath('.api', 'v' + visualConfig.apiVersion, 'schema.stringResources.json')),
+                    this._getValidationSchema(stringResourcesPath),
```
Comment (ignatvilesov) [inline lib/VisualBuilder.js]: Looks nice but we must include a unit test that covers this change and confirms that the code works properly

Comment (EugeneElkin) [reply lib/VisualBuilder.js]: Ok. I will add

File: Changelog.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 This page contains information about changes to the PowerBI Visual Tools (pbiviz).
 
+## pbiviz v1.8.1
+* FIX: Localization schema will be ignorred if API doesn't support localization
```
Comment (ignatvilesov) [inline Changelog.md]: Typing at `ignorred`. Please use `ignored` instead

File: spec/e2e/pbivizStartSpec.js [hunk 4]
```diff
@@ -77,6 +76,68 @@ describe("E2E - pbiviz start", () => {
         expect(error.message).toContain("Error: pbiviz.json not found. You must be in the root of a visual project to run this command");
     });
 
+    it("Should build visual with API 1.5 and check that is is started correctly (string resources doesn't exist)", (done) => {
```
Comment (ignatvilesov) [inline spec/e2e/pbivizStartSpec.js]: There's double `is` at `that is is`.

File: spec/e2e/pbivizStartSpec.js [hunk 5]
```diff
@@ -77,6 +76,68 @@ describe("E2E - pbiviz start", () => {
...
+                    done();
+                });
+            }
+        });
+        pbivizProc.stderr.on('data', (data) => {
+            throw new Error(data.toString());
+        });
+    });
+
+    it("Should build visual with API 1.6 and check that is is started correctly (string resources exists)", (done) => {
```
Comment (ignatvilesov) [inline spec/e2e/pbivizStartSpec.js]: The same typing mistake

**Pull request merged**